cmd/kops/update_cluster.go

```diff
@@ -65,6 +65,7 @@ type UpdateClusterOptions struct {
 	SSHPublicKey    string
 	MaxTaskDuration time.Duration
 	CreateKubecfg   bool
+	NodeUpSource    string
```

Comment:
It's used down below when ApplyClusterCmd is instantiated. I wasn't aware of `KOPS_BASE_URL`. Perhaps this is redundant. I can drop the commit from the PR if that's better.